packages/react-devtools-extensions/src/main.js

```diff
@@ -25,21 +25,25 @@ const LOCAL_STORAGE_SUPPORTS_PROFILING_KEY =
 
 const isChrome = getBrowserName() === 'Chrome';
 
-const cachedNetworkEvents = new Map();
+const cachedSources = new Map();
 
 // Cache JavaScript resources as the page loads them.
 // This helps avoid unnecessary duplicate requests when hook names are parsed.
 // Responses with a Vary: 'Origin' might not match future requests.
 // This lets us avoid a possible (expensive) cache miss.
 // For more info see: github.com/facebook/react/pull/22198
 chrome.devtools.network.onRequestFinished.addListener(
-  function onRequestFinished(event) {
-    if (event.request.method === 'GET') {
-      switch (event.response.content.mimeType) {
+  function onRequestFinished(request) {
+    if (request.request.method === 'GET') {
```

Comment:
nit: I think "event" is a better name. It is an event. That's why it was `event.request` (not `request.request`). Maybe we can change the name back? 😅

Reply:
Maybe we could also add some `__DEBUG__` logging to this PR (like I did in #22285, even though that one likely won't land)– just to make it clearer when we're getting in-memory cache hits vs misses?

Reply:
i named it request because that’s the chrome docs name it: https://developer.chrome.com/docs/extensions/reference/devtools_network/#event-onRequestFinished

it’s technically a [Request](https://developer.chrome.com/docs/extensions/reference/devtools_network/#type-Request) object, so i figured i’d name it more according to that, but if you feel strongly i can change it

Reply:
I _see_ 😮  `request.request` seems super goofy on their part for a naming choice.

Okay in that case, I guess it's fine to name it whatever you want 😅